.github/workflows/release.yaml

```
@@ -465,104 +465,48 @@ jobs:
       - name: Download data
         run: sh download_data.sh
 
-      - name: Install s3cmd
+      - name: Install s3cmd (staging)
         if: github.ref != 'refs/heads/release'
         uses: ./.ci/install-s3cmd
         with:
-          access-key: ${{ secrets.STAGING_ASSETS_BUCKET_ACCESS_KEY }}
-          secret-key: ${{ secrets.STAGING_ASSETS_BUCKET_SECRET_KEY }}
+          access-key: ${{ secrets.AI_ASSETS_STG_BUCKET_ACCESS_KEY }}
+          secret-key: ${{ secrets.AI_ASSETS_STG_BUCKET_SECRET_KEY }}
+
+      - name: Install s3cmd (production)
+        if: github.ref = 'refs/heads/release'
+        uses: ./.ci/install-s3cmd
+        with:
+          access-key: ${{ secrets.AI_ASSETS_PROD_BUCKET_ACCESS_KEY }}
+          secret-key: ${{ secrets.AI_ASSETS_PROD_BUCKET_SECRET_KEY }}
 
       - name: Upload assets to S3
-        if: github.ref != 'refs/heads/release'
+        id: s3-upload
         shell: bash
-        env:
-          BUCKET_URL: s3://xayn_ai_staging_assets
         run: |
           set -e
 
+          if [[ "${{ github.ref }}" = "refs/heads/release" ]]
+            then
+              export BUCKET_URL="s3://${{ secrets.AI_ASSETS_BUCKET_PROD }}"
+          else
+              export BUCKET_URL="s3://${{ secrets.AI_ASSETS_BUCKET_STG }}"
```

since the bucket keys have the word order `STG/PROD_BUCKET`, should we follow the same naming for the bucket url?
```suggestion
              export BUCKET_URL="s3://${{ secrets.AI_ASSETS_PROD_BUCKET }}"
          else
              export BUCKET_URL="s3://${{ secrets.AI_ASSETS_STG_BUCKET }}"
```

---

Already updated the secrets and 1Password as well! 🚀